processing/src/main/java/org/apache/druid/segment/writeout/FileWriteOutBytes.java

```diff
@@ -105,8 +110,8 @@ public void write(byte[] b, int off, int len) throws IOException
   @Override
   public long size() throws IOException
   {
-    flush();
-    return ch.size();
+    flushIfNeeded(0); // To avoid check the declared IOException never thrown error
```

Comment:
I think this is cleaner

```
  public long size()
  {
    return writeOutBytes;
  }
```

Reply:
Thanks for point out that. I added this line of code on purpose, or `The declared exception <code>IOException</code> is never thrown` error will block ci. And I attempt to remove `throws IOException` from method signature, but new error occurs! Do you have any suggestions?